content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,200 @@
+
+After closing the credential creation menu, the user can observe the prior created API Client Credentials by identifying the Credential Name.
+
+![](/img/application-credential-created-prior-shown.png "API Client Credentials are created")
+
+Clicking on the down arrow, user can generate the access token required to use the API.
+
+![](/img/api-client-credential-get-access-token.png "Time to obtain the Access Token")
+
+After clicking on the Generate Access Tokens button, the menu requires user to enter the Client Secret obtained from the API Client Credentials. This is the time to copy and paste the client secret from Credential Created menu so that the user can obtain the access token.
+
+![](/img/generate-access-token-with-secret.png "Use the client secret to generate Access Token")
+
+Copy the access token using using the "sheets" icon and store this access token to a safely recorded location.
+
+![](/img/access-token-created-and-close.png "Access Token Generated and Consumed")
+
+Afterward, the user can embed the access token to the REST API request in order to perform the HTTP method against the desired resource in order to obtain the response.  Note that the user must use the correct base-URL according to the region where the DSCC is deployed. Currently these are the base-URL and the corresponding region where the DSCC is deployed (November 2021)
+
+| DSCC Region  | base-URL                       |
+| ------------ | ------------------------------ |
+| EU Central   | https://eu1.data.cloud.hpe.com |
+| AP Northeast | https://jp1.data.cloud.hpe.com |
+| US West      | https://us1.data.cloud.hpe.com |
+
+#### Oops! What if I missed to copy the client secret for this instance of client credential?
+
+The client secret can be recreated inside the create credentials menu by clicking on the three dots at the bottom menu. Once the three dot menu opened, the user must click on the Reset Client Secret menu. This prior action will open the Credentials Created menu to show the new client secret for one time only. 
+
+![Reset the client secret](/img/reset-the-client-secret-in-client-credential-menu.png "resetting the client secret")
+
+#### Nice! Can you give me an example of using the access token?
+
+The access token is a long string of JSON Web Token that is signed using RS256 algorithm. Note that the access-token must be added into the header of with keyword "Authorization: Bearer <access-token in JWT>". This following example is based on curl tool, and it uses "https://scalpha-app.qa.cds.hpe.com" as base-URL. Note that this base-URL is DSCC testing-site only, please use one of the base-URL noted in the above table. The following example of the DSCC REST API request uses GET method for this resource /api/v1/audit-events to obtain lists of the audit-events available. Note the additional parameter with keyword "Authorization: Bearer" is added into the header of this REST API request.
```

Comment:
Add "for the" prior to "DSCC testing-site only". Add a period immediately thereafter and start the next sentence with "Please use one of the base-URLs noted..."